packages/maskbook/src/web3/UI/EthereumMaskBalanceButton.tsx

```diff
@@ -0,0 +1,34 @@
+import { Button, createStyles, makeStyles, Typography } from '@material-ui/core'
+import { useStylesExtends } from '../../components/custom-ui-helper'
+import { MaskbookIcon } from '../../resources/MaskbookIcon'
+
+const useStyles = makeStyles((theme) => {
+    return createStyles({
+        root: {
+            borderRadius: 16,
+            fontWeight: 300,
+            backgroundColor: '#1C68F3',
+            '&:hover, &:active': {
+                backgroundColor: '#1C68F3',
```

Comment:
is it dark/light mode ready?

Reply:
This is the blue color from the Mask icon.